.travis.yml

```diff
@@ -32,9 +32,9 @@ env:
     # AWS_REDSHIFT_USER
     - secure: "LIkY/ZpBXK3vSFsdpBSRXEsgfD2wDF52X8OZOlyBJOiZpS4y1/obj8b3VQABDPyPH95bGX/LOpM0vVM137rYgF0pskgVEzLMyZOPpwYqNGPf/d4BtQhBRc8f7+jmr6D4Hrox4jCl0cCKaeiTazun2+Y9E+zgCUDvQ8y9qGctR2k="
     # TEST_AWS_ACCESS_KEY_ID
-    - secure: "kFmNLBHXPCIMdbDtHpULFzQtQqWJOO5Q3PtHyvfXE9oHaVwmQUfNd2ygoH5qnKuHO66bfPC7GSY5cMERNIFXsdhDEbRYFj6B4OgcQsYmKAG6b3+qcFYq6htJqkeFbOZB8kUN4YOjlkZg1lPXp/RV0Ac1g72mEsm2CvIoCXmDMYg="
+    - secure: "es6ERiruS8vrv6Ur2GTs5aD3ULcfuGZGJWH/AJhjncVdC5+bkVg2mIMqFjh1sDXlQhUtLXNC9cpg9ZMTBSwn+xYdIr2u8iJZZX3kvmLwKOeR+wsWJpt5WbUPM4eIyZC9tHP9VlL42CJ2AxrHyiGSqSOl1jdMeFMadVUwkmiHg7M="
```

Comment:
Note to self: I should actually roll this change back, since I explicitly want to use credentials which contain backslashes to make sure that it will still work if those credentials are provided as part of the Hadoop configuration.